website/README.md

```diff
@@ -5,37 +5,21 @@ This website is built using [Docusaurus 2](https://docusaurus.io/), a modern sta
 ### Installation
 
 ```
-$ yarn
+$ pnpm i
```

Comment:
Is there a reason why we are choosing different workflows for our website and Node.js bindings?

Reply:
There is no specific reason. It can be the same.

Reply:
Let's use the same toolchain.